### ISSUE TITLE
TextTrackDisplay not working in Safari with overrideNative false + nativeTextTracks false

### ISSUE THREAD

**Issue by philipgiuliani** [videojs/video.js#7904]
### Description

When using Safari the TextTrackDisplay is not updated. It only updates when resizing the browser / player.

After manually setting a EventListener on the `texttrackchange`-Event (used here: https://github.com/videojs/video.js/blob/main/src/js/tracks/text-track-display.js#L105) I've noticed that the event is never triggered. The rest seems to work fine.

Using some hack like: `setInterval(() => this.player.textTrackDisplay.updateDisplay(), 100);` makes the captions update perfectly fine.

### Steps to reproduce

1. Example with Video.JS +HLS Video with captions
2. Set the following config to the player:
    ```
    html5: {
        vhs: {
            withCredentials: true,
            overrideNative: true,
        },
        nativeTextTracks: false,
        nativeAudioTracks: false,
        nativeVideoTracks: false,
     }
    ````
3. Start playback and select a caption language

**Expected**
Captions are displayed on the screen

**Actual result**
Captions are only displayed when resizing the browser or calling `this.player.textTrackDisplay.updateDisplay()` manually.

### Errors

No errors in the console

### What version of Video.js are you using?

v7.20.2

### Video.js plugins used.

http-streaming

### What browser(s) including version(s) does this occur with?

Safari 15.6.1

### What OS(es) and version(s) does this occur with?

macOS 12.5.1

**Comment by gkatsev** [comment]
I think this has been fixed in 7.20.3.

**Comment by philipgiuliani** [comment]
Seems to be fixed, thanks!
